osu.Game/Beatmaps/Formats/LegacyDecoder.cs

```diff
@@ -73,6 +73,9 @@ protected virtual void ParseLine(T output, Section section, string line)
             switch (section)
             {
                 case Section.Colours:
+                // osu!catch section only has colour settings
+                // so no harm in handling the entire section
```

Comment:
I'm not sure I agree with this comment... Extracting out a separate decoder like mania did might be overkill but I don't think this should be handled in the very base decoder, either. `LegacySkinDecoder` is probably a better place.